smartsim/_core/_cli/build.py

```diff
@@ -496,3 +499,17 @@ def configure_parser(parser: argparse.ArgumentParser) -> None:
         default=False,
         help="Build KeyDB instead of Redis",
     )
+
```

Comment:
VERY petty style thing: This is the only new line in this function between the `add_argument` calls and my OCD cannot handle it. Can you please remove or add a banner comment or something, lol?

Reply:
I think black did that on me. Let me remove and double-check

Reply:
Yep black cares little about your feelings. It's staying in

Reply:
How could black betray me like this 😢 . All well, seems reasonable to me